toolchain/sem_ir/stringify_type.cpp

```diff
@@ -441,6 +441,32 @@ auto StringifyTypeExpr(const SemIR::File& sem_ir, InstId outer_inst_id)
         }
         break;
       }
+      case CARBON_KIND(StructValue inst): {
+        auto field_values = sem_ir.inst_blocks().Get(inst.elements_id);
+        if (field_values.empty()) {
+          out << "{}";
+          break;
+        }
+        auto struct_type = sem_ir.types().GetAs<StructType>(inst.type_id);
```

Comment:
We build `StructValue`s of class type for class values too. We also build them for a few other types that "should be" class types -- for the values of `FunctionDecl`s, `ClassDecl`s, and `InterfaceDecl`s -- but those are all empty so don't get here.

I think we can handle the class type case with simply:
```suggestion
        auto struct_type = sem_ir.types().GetAs<StructType>(
            sem_ir.types().GetObjectRepr(inst.type_id));
```